### PR TITLE
fix: v1.0.1 hot fixes (ubuntu, version map issue, cosmovisor)

### PR DESCRIPTION
**Pull request by anikitinDSR**

### CODE REVIEW

File: .github/workflows/cleanup-actions.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
   
   delete-runs:
     name: "Delete old workflow runs"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (ankurdotb) [inline .github/workflows/cleanup-actions.yml:35]: You can keep this on latest, doesn't matter (we'll probably need to switch to Ubuntu 22.04 soon anyway)

File: .github/workflows/cleanup-cache-automatic.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
 
   cache-purge:
     name: "Purge Actions cache"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (ankurdotb) [inline .github/workflows/cleanup-cache-automatic.yml:15]: Keep on latest

File: .github/workflows/cleanup-cache-manual.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
 
   delete-caches:
     name: "Delete Actions caches"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (ankurdotb) [inline .github/workflows/cleanup-cache-manual.yml:15]: Keep on latest

File: .github/workflows/pull-request.yml [hunk 4]
```diff
@@ -19,7 +19,7 @@ concurrency:
 jobs:
   lint-pr:
     name: "PR format check"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (ankurdotb) [inline .github/workflows/pull-request.yml:22]: Keep on latest

File: .goreleaser.yml [hunk 5]
```diff
@@ -60,11 +60,8 @@ archives:
         linux: Linux
         amd64: x86_64
     format: tar.gz
-    wrap_in_directory: true
+    wrap_in_directory: false
```
Comment (ankurdotb) [inline .goreleaser.yml:63]: Yep all of these goreleaser changes look good to me

**Review by ankurdotb — APPROVED** [review]
Looks good, some of these can remain as latest (will keep it in sync with other repos)

**Review by Eengineer1 — APPROVED** [review]

**Pull request merged**